src/storage/src/hummock/shared_buffer/shared_buffer_batch.rs

```diff
@@ -97,6 +97,10 @@ impl SharedBufferBatch {
     pub fn epoch(&self) -> u64 {
         self.epoch
     }
+
+    pub fn len(&self) -> usize {
```

Comment:
Why prefer len() instead of size()? Is it a convention of Rust?

Reply:
Yep.